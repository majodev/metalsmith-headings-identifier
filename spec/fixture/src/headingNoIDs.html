<h1>1 heading</h1>
<h2>2 heading</h2>
<h3>3 heading</h3>
<h4>4 heading</h4>
<h5>5 heading</h5>
<h6>6 heading</h6>