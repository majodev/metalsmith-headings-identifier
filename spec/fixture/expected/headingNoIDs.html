<h1><a class="heading-anchor" href="#1-heading"><span></span></a>1 heading</h1>
<h2><a class="heading-anchor" href="#2-heading"><span></span></a>2 heading</h2>
<h3><a class="heading-anchor" href="#3-heading"><span></span></a>3 heading</h3>
<h4><a class="heading-anchor" href="#4-heading"><span></span></a>4 heading</h4>
<h5><a class="heading-anchor" href="#5-heading"><span></span></a>5 heading</h5>
<h6><a class="heading-anchor" href="#6-heading"><span></span></a>6 heading</h6>